packages/node_modules/@webex/plugin-rooms/test/integration/spec/rooms.js

```diff
@@ -200,72 +206,77 @@ describe('plugin-rooms', function () {
     describe('#listWithReadStatus()', () => {
       // Reminder: we can't run the next two creates in parallel because some of
       // the tests rely on ordering.
-      let room1, room2, spark2, actor2;
+      let room1, room2, user2;
 
       before(() => testUsers.create({count: 1})
         .then(([user]) => {
-          spark2 = new CiscoSpark({credentials: user.token});
-
-          return spark2.people.get('me')
-            .then((person) => {
-              actor2 = person;
-              debug('SDK User (Actor) for tests:');
-              debug(`- name: ${actor.displayName}`);
-              debug(`-   id: ${actor2.id}`);
-            });
+          user2 = user;
```

Comment:
It looks like you only need one user. If you don't want to use the cool name `spock` 😎 , do 

```
...
.then([u]) => {
  user = u;
...
```

Reply:
nm, I see why you did this.